### PR TITLE
bugfix(fb custom audience): lowerCase() error for number data

### PR DESCRIPTION
**Pull request by shrouti1507**
## Description of the change

`.toLowerCase` function operates only strings. In case, if number is entered, it throws error. That is why inputs are converted to string, before the above function starts to operate.

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by utsabc** [comment]
@shrouti1507 for numeric values `toString` will still fail if you do lowercasing on string numbers. You have to validate numeric values and before performing the operation. For numeric identifier lets skip that identifier as throwing error will cancel the batch of identifiers
cc @Vikas26021999

File: v0/destinations/fb_custom_audience/transform.js [hunk 1]
```diff
@@ -111,27 +114,30 @@ const ensureApplicableFormat = (userProperty, userInformation) => {
...
+      updatedProperty = userInformation.toString().toLowerCase();
       break;
     case "ZIP":
-      userInformationTrimmed = userInformation.replace(/\s/g, "");
+      userInformationTrimmed = userInformation.toString().replace(/\s/g, "");
       updatedProperty = userInformationTrimmed.toLowerCase();
       break;
     case "ST":
     case "CT":
       updatedProperty = userInformation
```
Comment (ItsSudip) [inline v0/destinations/fb_custom_audience/transform.js:139]: Apart from calling toString so many times we can directly make `userInformation` a string where we are calling `ensureApplicableFormat` function.

Comment (shrouti1507) [reply v0/destinations/fb_custom_audience/transform.js:139]: yeah we can 😅

**Review by utsabc — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Comment by shrouti1507** [comment]
> @shrouti1507 for numeric values `toString` will still fail if you do lowercasing on string numbers. You have to validate numeric values and before performing the operation. For numeric identifier lets skip that identifier as throwing error will cancel the batch of identifiers cc @Vikas26021999

This part is solved. As we saw it was not failing.

**Review by Vikas26021999 — APPROVED** [review]
LGTM!

**Pull request merged**